UnityProject/Assets/Scripts/Items/ItemAttributesV2.cs

```diff
@@ -4,6 +4,8 @@
 using Mirror;
 using AddressableReferences;
 using Systems.Clothing;
+using NaughtyAttributes;
```

Comment:
what did you add these namespaces for?

Reply:
I added them when I was working and forgot to delete it. Should be solved in the latest commits